### DISCUSSION TITLE
Compatibility with Nuxt 2?

### DISCUSSION THREAD

**Discussion by aniash29-cr** [ivodolenc/nuxt-font-loader#17] [Q&A]
Tried to use the updated package with one of the projects on Nuxt 2. Does not seem to work, probably because of the module configuration?

No plans for backward compatibility?

**Comment by ivodolenc** [comment]
> Tried to use the updated package with one of the projects on Nuxt 2. Does not seem to work, probably because of the module configuration?

Hi, sorry for the late reply.

The last `nuxt-font-loader`  version that supports Nuxt 2 is `1.1.5`.

All versions after that are designed for Nuxt 3 and will not work for Nuxt 2.

Here is the [docs](https://www.npmjs.com/package/nuxt-font-loader/v/1.1.5) for Nuxt 2. 

> No plans for backward compatibility?

At this point, no.
